portal/factories/app.py

```diff
@@ -171,6 +171,7 @@ def configure_jinja(app):
     app.jinja_env.globals.update(app_text=app_text)
     app.jinja_env.globals.update(ROLE=ROLE)
     app.jinja_env.globals.update(INTERVENTION=INTERVENTION)
+    app.jinja_env.globals.update(WTF_CSRF_TIME_LIMIT=app.config.get("WTF_CSRF_TIME_LIMIT"))
```

Comment:
I'm wary about adding a global template var that may only be needed in a handful of templates. 

Do you expect you'll need to use this template var in more templates in the future? Would it make sense to explicitly pass it to the 2 changed templates in this PR?

Reply:
The two changed templates are the base templates from which many other templates are **extending** from.  I _**think**_ this approach is cleaner than passing the variable to all the ones that need them.